.github/stale.yml

```diff
@@ -0,0 +1,21 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 30
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - do-not-stale
```

Comment:
nit picky, but I don't like this label name. Can you think of something better e.g. `do-not-go-stale`?

Reply:
maybe also add the `todo-later` label to `exemptLabels` ?

> nit picky, but I don't like this label name. Can you think of something better e.g. do-not-go-stale

I do understand that you don't like the wording but I find that `do-not-stale` is already pretty long and `do-not-go-stale` is even longer and thereby uses up a lot of space which I don't like. Can we come up with something shorter?

Reply:
> Can we come up with something shorter?

Like ?

Reply:
Yes, `pinned` is good IMHO.

Reply:
Agreed.